### PR TITLE
PDF guides from markdown files

### PR DESCRIPTION
**Pull request by pshevtsov**
This PR introduces the following changes:

- Set of tools (`make`, `latex`, `pandoc`) and their configurations to produce PDF guides based on the existing Markdown files.

## How to build PDF files

Follow the installation instructions in the `README.md` file, then run `make -j8 book`

## How to create a new PDF guide

Create a new Markdown file (e.g. `new-guide.md`) in the `book` directory (use `discover.md` as an example). Write or include the contents, run `make -j8 book`. The new file (in our case `new-guide.pdf`) would be created in the `book` directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,30 @@ To run [jekyll](https://jekyllrb.com/) and start hacking run:
 
 
 You should see the rendered site on http://localhost:4000
+
+## PDF generation
+
+You need to have [`pandoc`](https://pandoc.org/installing.html) and LaTeX
+(`lualatex`) installed. We suggest to use
```
Comment (jmariomejiap) [inline README.md:32]:
```suggestion
(`lualatex`) installed. We suggest using
```

File: README.md [hunk 2]
```diff
@@ -25,3 +25,30 @@ To run [jekyll](https://jekyllrb.com/) and start hacking run:
...
+- `everypage`
+- `fancyhdr`
+- `luatexbase`
+- `noto`
+- `pgf`
+- `selnolig`
+- `sourcecodepro`
+- `tcolorbox`
+
+The above packages could be easily installed by running `./book/deps.sh` script.
```
Comment (jmariomejiap) [inline README.md:48]: would the script work on other systems that are not Linux?

File: book/template-blocks.lua [hunk 3]
```diff
@@ -0,0 +1,138 @@
+-- template-blocks.lua - transform Jekyll template blocks
+
```
Comment (jmariomejiap) [inline book/template-blocks.lua:2]: This Lua code is pretty nice 👍 .

File: pages/tidal tools/autocomplete.md [hunk 4]
```diff
@@ -58,7 +58,7 @@ brew install bash-completion@2
 
 ```
 
-{% include note.html content="Follow the “caveats” section of brew’s output to add the appropriate bash completion path to your local .bashrc. to enable bash completion for any new shell session" %}
+{% include note.html content="Follow the *caveats* section of `brew`’s output to add the appropriate bash completion path to your local `.bashrc` to enable bash completion for any new shell session" %}
```
Comment (jmariomejiap) [inline pages/tidal tools/autocomplete.md:61]: nice, good to have those ``. I didn't notice they were missing

File: pages/tidal tools/getstarted.md [hunk 5]
```diff
@@ -8,11 +8,10 @@ permalink: tidal-tools.html
...
 ---
 
-{% include note.html content="
-This page describes general installation and configuration instructions for
-Tidal Tools. If you are looking for specific AWS usage instractions please
-refer to [Getting Started with Tidal Tools on AWS](tidal-tools-aws.html) guide.
-" %}
+{% include note.html content="This page describes general installation and
+configuration instructions for Tidal Tools. If you are looking for specific AWS
+usage instractions please refer to [Getting Started with Tidal Tools on
```
Comment (jmariomejiap) [inline pages/tidal tools/getstarted.md:13]:
```suggestion
usage instructions please refer to [Getting Started with Tidal Tools on
```

**Review by jmariomejiap — CHANGES_REQUESTED** [review]
It looks good to me @pshevtsov. I added a few suggestions/comments.

File: README.md (hunk 2 above)
Comment (pshevtsov) [reply README.md:48]: No, it would not. More than that, it won't work on any Linux other than Debian Stable.

Comment (jmariomejiap) [reply README.md]: @pshevtsov  Do you think it is worth mentioning?

**Review by jmariomejiap — APPROVED** [review]

File: book/plan.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (prees1) [inline book/plan.md:1]: Is there a use for this guide, the `developers` and `assess` ones to be on master considering they are empty and still a work-in-progress before anyone can use them?

File: book/deps.sh [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+# Install pandoc, we use version 2.11.2
+wget https://github.com/jgm/pandoc/releases/download/2.11.2/pandoc-2.11.2-1-amd64.deb
+sudo dpkg -i pandoc-2.11.2-1-amd64.deb
+
+# Install tinytex
+wget -qO- "https://yihui.org/tinytex/install-bin-unix.sh" | sh
+
+# Install needed latex packages for our use
+tlmgr install \
+  environ \
```
Comment (prees1) [inline book/deps.sh:12]: Nice! Curious how you managed to _decrease_ the dependencies here from the previous larger list of deps.

**Review by prees1 — COMMENTED** [review]
Looks great overall to me.

File: book/plan.md (hunk 6 above)
Comment (pshevtsov) [reply book/plan.md:1]: I'm afraid, I don't understand your question. Are you asking about `plan` guide in particular, or all of the guides except `discover`?

File: book/deps.sh (hunk 7 above)
Comment (pshevtsov) [reply book/deps.sh:12]: Previously we used some 3rd-party template with lots of dependencies, which we hadn't actually used. This time we use the standard template with some minor additions (e.g. `tcolorbox`, `fancyhdr`, etc)

File: book/plan.md (hunk 6 above)
Comment (prees1) [reply book/plan.md:1]: Yea all three (except `discover`), they all look empty and still WIP

Comment (pshevtsov) [reply book/plan.md]: Removed WIP files in 121b05f

**Review by jmariomejiap — APPROVED** [review]
Looks ready to 🚀

**Pull request merged**